lib/magma/attribute.rb

```diff
@@ -1,14 +1,25 @@
 class Magma
   class Attribute
     DISPLAY_ONLY = [:child, :collection]
-    EDITABLE_OPTIONS = [:description, :display_name, :format_hint]
 
-    attr_reader :name, :loader, :validation, :format_hint, :unique, :index, :restricted
+    EDITABLE_OPTIONS = [
+      :description,
+      :display_name,
+      :format_hint,
+      :hidden,
+      :index,
+      :loader,
+      :read_only,
+      :restricted,
+      :unique
+    ]
+
+    attr_reader :name, :loader, :validation, :format_hint, :unique, :index, :restricted, :description
 
     class << self
       def options
-        [:description, :display_name, :hide, :readonly, :unique, :index, :validation,
-:format_hint, :loader, :link_model, :restricted, :desc ]
+        [:description, :display_name, :hidden, :read_only, :unique, :index, :validation,
```

Comment:
`hide` and `readonly` were renamed to `hidden` and `read_only` per https://github.com/mountetna/magma/issues/99#issuecomment-621507560.